ext/pdo_sqlite/config.m4

```diff
@@ -66,6 +66,10 @@ if test "$PHP_PDO_SQLITE" != "no"; then
   PHP_CHECK_LIBRARY(sqlite3,sqlite3_key,[
     AC_DEFINE(HAVE_SQLITE3_KEY,1, [have commercial sqlite3 with crypto support])
   ])
+  PHP_CHECK_LIBRARY(sqlite3,sqlite3_load_extension,
+    [],
+    [AC_DEFINE(SQLITE_OMIT_LOAD_EXTENSION, 1, [have sqlite3 with extension support])
```

Comment:
This is a bit odd, we'd usually go for a positive define instead. `HAVE_SQLITE3_LOAD_EXTENSION` would be typical.